### DISCUSSION TITLE
How to uninstall a font in v4.0.2

### DISCUSSION THREAD

**Discussion by AlexHolly** [wixtoolset/issues#7728] [Questions] [answered]
How to uninstall a font (v4.0.2)?

When I "uninstall" the Font is still there. I tried the following:

```
            <Component Id="FontComponent" Guid="5d37a387-cc01-406e-9689-cd146c4cc6af">
                <File Id="Font.ttf" Name="Font.ttf" Source="$(var.SolutionDir)..\_Setup\Font.ttf" TrueType="yes" KeyPath="yes" />
                <RemoveFile Id="RemoveFont" On="install" Name="Font.ttf" />
            </Component>
```

`On="uninstall"` is also not working.

**Comment by robmen** [comment]
The `RemoveFile` is not necessary. Uninstalling the `Component` should remove the `Font`. An uninstall log file will show you what's happening.

**Comment by AlexHolly** [comment, marked as answer]
It works now without the `RemoveFile` line. The only reason I can think of why it did not work are multiple installers with the same font.
Thank you.

**Reply by chrpai** [reply]
Anytime I see this I assume dirty machine.  Test on a clean VM and it should work.
